lib/segment/analytics/version.rb

```diff
@@ -1,5 +1,5 @@
 module Segment
   class Analytics
-    VERSION = '2.2.8.pre'
```

Comment:
@nd4p90x Do you think we need a major version bump here, given we dropped support for older versions of Ruby?

Reply:
I was updating the version file to match the actual history, we moved out of -pre in December 2020

Reply:
I update the minor version with history